### PR TITLE
Fixed exception handling, updated dependencies.

### PR DESCRIPTION
**Pull request by AntonCroner**
Due to an issue in cloudly-http 0.2 http.Response.create the serialization of exceptions was broken.

### CODE REVIEW

File: Router.ts [hunk 1]
```diff
@@ -95,6 +94,6 @@ export namespace Router {
 		readonly origin: (string | RegExp)[]
 		readonly allowHeaders: (keyof http.Request.Header | string)[]
 		readonly middleware: http.Middleware
-		readonly catch: boolean | ((exception: unknown) => http.Response.Like)
+		readonly catch: boolean | ((exception: unknown) => http.Response)
```
Comment (simonmika) [inline Router.ts:97]: No, we don't want this change.

File: Router.ts [hunk 2]
```diff
@@ -31,13 +31,12 @@ export class Router<T> {
...
+						? this.options.catch(error)
+						: {
 								status: 500,
-								type: "unknown error",
-								error: "exception",
-								description: (typeof error == "object" && error && error.toString()) || undefined,
-						  })
+								header: { contentType: "text/plain" },
+								body: (error && typeof error == "object" && error.toString()) || undefined,
+						  }
```
Comment (simonmika) [inline Router.ts:39]: We explicitly want the error message to be a JSON compatible with a gracely error.

**Review by simonmika — CHANGES_REQUESTED** [review]
Fix the real error instead of introducing new faults. Ask for help if needed.

File: Router.ts (hunk 2 above)
Comment (AntonCroner) [reply Router.ts]: Reverted.

Comment (AntonCroner) [reply Router.ts]: Reverted.

**Pull request closed without merging**